in_container_scripts/run_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eufo pipefail
+
+# test pydrake install
+if pip list 2>/dev/null | grep -q pydrake &>/dev/null; then
+    echo "pydrake not found in pip list"
+    exit 1
+else
+    python3 -c "import pydrake" || (echo "pydrake import failure" && exit 1)
```

Comment:
Better to use an `if ... else` here. The `exit 1` doesn't directly exit the top-level shell; only the subshell (which then exits in turn because a sub-command exited with failure)